### PR TITLE
Add molecule test to ovn_bgp_agent component

### PR DESCRIPTION
**Pull request by fernandoroyosanchez**
Add molecule test to ovn_bgp_agent component

### CODE REVIEW

**Comment by fernandoroyosanchez** [comment]
recheck

**Comment by fernandoroyosanchez** [comment]
recheck

**Review by rebtoor — CHANGES_REQUESTED** [review]
You should add this molecule scenario to zuul (see `zuul.d` folder in the root of the repo) to be executed.

File: roles/edpm_ovn_bgp_agent/tasks/configure.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@
         recurse: true
         patterns:
           - "*ovn.conf"
+          - "*ovn-bgp-agent*conf"
```
Comment (luis5tb) [inline roles/edpm_ovn_bgp_agent/tasks/configure.yml:40]: this is not needed, please remove

File: roles/edpm_ovn_bgp_agent/tasks/run.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 - name: Run ovn_bgp_agent container
   ansible.builtin.include_role:
-    name: edpm_container_standalone
+    name: osp.edpm.edpm_container_standalone
```
[the remark this answers is not in the document]
Comment (fernandoroyosanchez) [reply roles/edpm_ovn_bgp_agent/tasks/run.yml:24]: done

File: roles/edpm_ovn_bgp_agent/tasks/configure.yml (hunk 1 above)
Comment (fernandoroyosanchez) [reply roles/edpm_ovn_bgp_agent/tasks/configure.yml]: done!

**Comment by fernandoroyosanchez** [comment]
> You should add this molecule scenario to zuul (see `zuul.d` folder in the root of the repo) to be executed.

already added! thx for the review!!!!

**Pull request merged**